testing/chain.go

```diff
@@ -317,7 +318,7 @@ func (chain *TestChain) SendMsgs(msgs ...sdk.Msg) (*sdk.Result, error) {
 	// ensure the chain has the latest time
 	chain.Coordinator.UpdateTimeForChain(chain)
 
-	_, r, err := simapp.SignAndDeliver(
+	_, r, err := simtestutil.SignCheckDeliver(
```

Comment:
this needs to be undone. When we deliver a tx, we need the ibc testing package to call end block and commit(). This is because we need to track information about the resulting validator set changes. Note how [here](https://github.com/cosmos/ibc-go/blob/main/testing/chain.go#L289) we use the `res` of the end block to update the tracked validator set. I'm sure there are ways we could adjust the code on either end to make the reuse work, but I think better to not spend too much time on that now